### DISCUSSION TITLE
Unable to install sbfspot-config...

### DISCUSSION THREAD

**Discussion by Femtohenry** [SBFspot/sbfspot-config#62] [Q&A]
Hello
as a dumb user of linux, i am not able to install "sfbspot-config".
It aborts with the error message "Failed to get username".
What is the reason for this? What to do?
Thanks for an answer

**Comment by sbf-** [comment]
How do you start sbfspot-config? Show the output pls

What is the output of `who -m`

**Reply by Femtohenry** [reply]
Thanks for the answer.
Have just started with linux.

the output is as follows:

```
root@raspberrypi:~# curl -s https://raw.githubusercontent.com/sbfspot/sbfspot-config/master/sbfspot-config|sudo bash
SBFspot Configuration Tool V1.3.4
Running on Raspberry Pi 3 Model B Plus Rev 1.3
Checking whiptail... Installed.
architecture=armhf
Raspbian GNU/Linux 11 (bullseye)
Failed to get username
root@raspberrypi:~#
root@raspberrypi:~#
root@raspberrypi:~# who -m
pi       pts/0        2023-03-31 13:41 (192.168.0.146)
root@raspberrypi:~#
```

**Reply by sbf-** [reply]
OK, I see... you're logged in as `root`

Either login as user `pi` or remove `sudo` from the command line like this:
`curl -s https://raw.githubusercontent.com/sbfspot/sbfspot-config/master/sbfspot-config|bash`

**Reply by Femtohenry** [reply]
That was it. Thank you for help!
Probably more questions will come up...
